src/platforms/react-native/manual-setup/codepush.mdx

```diff
@@ -70,12 +70,13 @@ sentry-cli react-native appcenter {APP} ios ./build/CodePush --deployment {DEPLO
 ```
 
 For non-macOS devices building for iOS, you will need to pass extra parameters that can't be computed automatically (`--bundle-id` and `--version-name`).
+`--version-name` comes from appcenter and has the `{}@{}+codepush:{}` pattern (see [here](https://github.com/getsentry/sentry-cli/blob/e08e7c31d93a29797edf7d1e5ec42ee0453ecc49/src/utils/appcenter.rs#L191-L194)), it should be the value within the 2nd `{}`.
```

Comment:
I'd rather prefer to mention that version name is the equivalent of `CFBundleShortVersionString` on iOS.

Reply:
I'll update that